psi4/driver/wrapper_database.py

```diff
@@ -229,16 +229,13 @@ def database(name, db_name, **kwargs):
     kwargs.pop('molecule', None)
 
     # Paths to search for database files: here + PSIPATH + library + PYTHONPATH
-    psidatadir = core.get_datadir()
-    #nolongerpredictable psidatadir = __file__ + '/../..' if psidatadir is None else psidatadir
-    libraryPath = ':' + os.path.abspath(psidatadir) + '/databases'
-    driver_loc = os.path.dirname(os.path.abspath(__file__))
-    dbPath = os.path.abspath('.') + \
-        ':' + ':'.join([os.path.abspath(x) for x in os.environ.get('PSIPATH', '').split(':')]) + \
-        libraryPath + \
-        ':' + driver_loc  # so the databases can "import qcdb"
-
-    sys.path = [sys.path[0]] + dbPath.split(':') + sys.path[1:]
+    dbPaths = []
+    dbPaths.append(os.getcwd())
+    dbPaths.extend(os.environ.get('PSIPATH', '').split(os.path.pathsep))
+    dbPaths.append(os.path.join(core.get_datadir(), 'databases'))
+    dbPaths.append(os.path.dirname(__file__))
+    dbPaths = list(map(os.path.abspath, dbPaths))
+    sys.path = dbPaths + sys.path
```

Comment:
I would be a bit hesitant to mess with `sys.path[0]` even if you carefully handle it. Can we do a `sys.path[1:1] = db_paths`?

Reply:
Done!